service/server.py

```diff
@@ -25,6 +25,8 @@
 POSTCODE_REGEX = re.compile(address_utils.BASIC_POSTCODE_REGEX)
 NOF_SECS_BETWEEN_LOGINS = 1
 LOGGER = logging.getLogger(__name__)
+INCLUDE_TIME = True
```

Comment:
It would be good to include some context in the names of these two fields, so that they're self-explaining.